src/main/java/org/openlmis/referencedata/domain/Order.java

```diff
@@ -7,6 +7,7 @@
 import javax.persistence.*;
 import java.math.BigDecimal;
 import java.time.LocalDateTime;
+import java.util.Set;
 
 @Entity
 @Table(name = "orders")
```

Comment:
We're going to go with plural table names to avoid most reserved words in SQL.  e.g. order.  I threw up a style guide as a working document in the service template repository.  Lets collaborate on those service conventions there.